### PR TITLE
KUNPRO-1204 Review 2 0 Change footer only for Review 2 0

### PR DESCRIPTION
**Pull request by dennisat**

### CODE REVIEW

File: packages/kununu-footer/index.jsx [hunk 1]
```diff
@@ -61,10 +67,10 @@ export default class Footer extends Component { // eslint-disable-line
       <footer
         role="banner"
         id="footer"
-        className={`navbar-default ${styles.footer}`}
+        className={`navbar-default ${styles.footer} ${simpleMobile ? styles.simpleMobile : ''}`}
```
Comment (codingcircus) [inline packages/kununu-footer/index.jsx]: please use the classnames package here

File: packages/kununu-footer/index.jsx [hunk 2]
```diff
@@ -84,7 +90,6 @@ export default class Footer extends Component { // eslint-disable-line
               </div>
               ),
             )}
-
             {tuv ?
```
Comment (codingcircus) [inline packages/kununu-footer/index.jsx]: you could write it like tuv && <div ...>, then you don't need the ternary fallback

Comment (kaytcat) [reply packages/kununu-footer/index.jsx]: This will return false in the dom. You can only do this if you do something like tuv === 1 or something.

Comment (codingcircus) [reply packages/kununu-footer/index.jsx]: 🤔

File: packages/kununu-footer/index.jsx (hunk 1 above)
Comment (kaytcat) [reply packages/kununu-footer/index.jsx]: Since we are not yet using the classNames package in client-gear I am not sure I want to introduce it with such small ternary. I think we should bring it in only once we really need it.. if for instance we have complex classes being added.

File: packages/kununu-footer/index.jsx (hunk 2 above)
Comment (dennisat) [reply packages/kununu-footer/index.jsx]: Kaitlin is right here, if `tuv` is false it will return false on the dom.
it should be like this `{tuv && <div /> || null}` where is the same like `{tuv ? <div /> : null}`.
I have seen "false" text in the tests so please be carefull with this.
Also I have seen empty lines in the tests (where they are looking like bugs for me), these lines are generated from the returned empty strings, as it happened here. Null should be returned in this case, it is accepted by React (not undefined). The same happens in the dom, not only this the tests. So :child selector might be confused with these empty strings.
